src/tree/toplevel.rs

```diff
@@ -94,6 +97,24 @@ impl<T: ToplevelNodeBase> ToplevelNode for T {
         data.parent.set(Some(parent.clone()));
         data.is_floating.set(parent.node_is_float());
         self.tl_set_workspace(&parent.cnode_workspace());
+
+        for handle in data.manager_handles.borrow().lock().values() {
+            if let Some(parent_toplevel) = data.parent.get().unwrap().node_toplevel() {
+                if let Some(client) = data.parent.get().unwrap().node_client() {
+                    let parent_handles = parent_toplevel.tl_data().manager_handles.borrow().lock();
+                    let parent_handle = parent_handles
+                        .iter()
+                        .find(|k| k.0.0 == client.id)
+                        .map(|k| k.1);
+                    if let Some(parent_handle) = parent_handle {
+                        handle.send_parent(parent_handle.clone());
+                        handle.send_done();
+                    }
+                }
+            }
+
+            handle.send_done();
+        }
```

Comment:
This doesn't do anything since a toplevel advertised through this protocol can never be the child of another toplevel.